### PR TITLE
[COMP] Ann Arbor

### LINKED ISSUE #454: [COMP] Ann Arbor

### PR DESCRIPTION
**Pull request by WindowsVistaisCool**
End of Ann Arbor
Closes #454

### CODE REVIEW

File: src/main/java/frc/robot/command/PathToPose.java
Comment (WindowsVistaisCool) [inline src/main/java/frc/robot/command/PathToPose.java:1]: why is drivetrain and controller being added here if they are never being used

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 1]
```diff
@@ -59,31 +53,41 @@ public Climber() {
...
+
+        // softLimitConfR.ForwardSoftLimitEnable = true;
+        // softLimitConfR.ForwardSoftLimitThreshold = ClimbConstants.MAX_HEIGHT;
+
+        // hardLimitConf.ReverseLimitEnable = false;
 
-        climbMotorL.applyConfig(climbMotorL.getConfig().withFeedback(sensorConf).withSoftwareLimitSwitch(softLimitConf));
-        climbMotorR.applyConfig(climbMotorR.getConfig().withFeedback(sensorConf).withSoftwareLimitSwitch(softLimitConf));
+        // climbMotorL.applyConfig(climbMotorL.getConfig().withFeedback(sensorConf).withSoftwareLimitSwitch(softLimitConfL).withHardwareLimitSwitch(hardLimitConf));
+        // climbMotorR.applyConfig(climbMotorR.getConfig().withFeedback(sensorConf).withSoftwareLimitSwitch(softLimitConfR).withHardwareLimitSwitch(hardLimitConf));
```
Comment (WindowsVistaisCool) [inline src/main/java/frc/robot/subsystems/Climber.java:72]: sensorConfs not being applied?

**Review by WindowsVistaisCool — COMMENTED** [review]
have fun with all the formatting changes :)

**Review by HeeistHo — APPROVED** [review]
from what i see, smart climb needs to be rewritten and (according to kyle) so does path to pose.

wowie!

lots of cleanup needs to be done too

wowie the code is code 👍

**Pull request merged**